File: cpp/operadora.cpp
#include<iostream>
#include<iomanip>

using namespace std;

int main(){
    int minutos;
    double valor;

    cout << "Digite a quantidade de minutos: ";
    cin >> minutos;

    valor = 50.0;

    if(minutos>100){
        minutos = minutos - 100;
        valor = valor + minutos * 2.0;
    }

    cout << fixed << setprecision(2);
    cout << "Valor a pagar: R$ " << valor << endl;

	return 0;
}

File: java/operadora.java
import java.util.Locale;
import java.util.Scanner;

public class operadora {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int minutos;
	    double valor;

	    System.out.print("Digite a quantidade de minutos: ");
	    minutos = sc.nextInt();

	    valor = 50.0;

	    if (minutos>100) {
	        valor = valor + (minutos - 100) * 2.0;
	    }
		
	    System.out.printf("Valor a pagar: R$ %.2f\n", valor);

		sc.close();
	}
}
